Makefile

```diff
@@ -167,9 +190,21 @@ release-upload-cluster-controller: release-cluster-controller upload-artifacts
 upload-artifacts:
 	controllers/build/upload_artifacts.sh $(TAR_PATH) $(ARTIFACTS_BUCKET) $(PROJECT_PATH) $(CODEBUILD_BUILD_NUMBER) $(CODEBUILD_RESOLVED_SOURCE_VERSION)
 
-.PHONY: cluster-controller-binaries
-cluster-controller-binaries:
-	controllers/build/create_binaries.sh $(GOLANG_VERSION) $(BINARY_NAME) $(IMAGE_REPO) $(IMAGE_TAG)
+PLATFORMS = linux-amd64 linux-arm64
```

Comment:
Lets define this above and make ?= just in case we want to override it, that also fits more our standard pattern with our makefiles.